pyproject.toml

```diff
@@ -132,6 +131,44 @@ authors = [
   { name="C. Titus Brown", orcid="0000-0001-6001-2677" },
 ]
 
+maintainers = [
+  { name="Luiz C. Irber, Jr", email="luiz@sourmash.bio", orcid="0000-0003-4371-9659" },
+  { name="C. Titus Brown", email="titus@idyll.org", orcid="0000-0001-6001-2677" },
+  { name="N. Tessa Pierce-Ward", email="ntpierce@ucdavis.edu", orcid="0000-0002-2942-5331" },
+]
+
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Environment :: Console",
+  "Environment :: MacOS X",
+  "Intended Audience :: Science/Research",
+  "License :: OSI Approved :: BSD License",
+  "Natural Language :: English",
+  "Operating System :: POSIX :: Linux",
+  "Operating System :: MacOS :: MacOS X",
+  "Programming Language :: Rust",
+  "Programming Language :: Python :: 3.13",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.11",
+  "Topic :: Scientific/Engineering :: Bio-Informatics",
+]
+
+dependencies = [
+  "screed>=1.1.3,<2",
+  "cffi>=1.14.0",
+  "numpy",
+  "matplotlib",
+  "scipy",
+  "deprecation>=2.0.6",
+  "cachetools>=4,<6",
+  "bitstring>=3.1.9,<5",
+]
+
+requires-python = ">=3.11"
```

Comment:
Did this get duplicated on a merge? It should be under the `[project]` section only, not `[tool.sourmash.metadata]`
```suggestion
```